cmd/observe/observe.go

```diff
@@ -177,14 +180,53 @@ more.`,
 
 	observerCmd.Flags().Var(filterVar(
 		"verdict", ofilter,
-		fmt.Sprintf("Show only flows with this verdict [%v, %v]", pb.Verdict_FORWARDED, pb.Verdict_DROPPED)))
+		fmt.Sprintf(
+			"Show only flows with this verdict [%v, %v, %v]",
+			pb.Verdict_FORWARDED, pb.Verdict_DROPPED, pb.Verdict_ERROR),
+	))
+	observerCmd.RegisterFlagCompletionFunc("verdict", func(_ *cobra.Command, _ []string, _ string) ([]string, cobra.ShellCompDirective) {
+		return []string{
+			pb.Verdict_FORWARDED.String(),
+			pb.Verdict_DROPPED.String(),
+			pb.Verdict_ERROR.String(),
+		}, cobra.ShellCompDirectiveDefault
+	})
 
 	observerCmd.Flags().Var(filterVar(
 		"http-status", ofilter,
 		`Show only flows which match this HTTP status code prefix (e.g. "404", "5+")`))
+	observerCmd.RegisterFlagCompletionFunc("http-status", func(_ *cobra.Command, _ []string, _ string) ([]string, cobra.ShellCompDirective) {
+		httpStatus := []string{
+			"100", "101", "102", "103",
+			"200", "201", "202", "203", "204", "205", "206", "207", "208",
+			"226",
+			"300", "301", "302", "303", "304", "305", "307", "308",
+			"400", "401", "402", "403", "404", "405", "406", "407", "408", "409",
+			"410", "411", "412", "413", "414", "415", "416", "417", "418",
+			"421", "422", "423", "424", "425", "426", "428", "429",
+			"431",
+			"451",
+			"500", "501", "502", "503", "504", "505", "506", "507", "508",
+			"510", "511",
```

Comment:
http package has these defined as `http.Status*`

Reply:
Yes, as a list of `const`. Not sure using them would make the code more readable and they'd have to be turned to string.

Reply:
Fair enough.

A helper function to expand `(400, 418)` into a list of strings from 400 to 418, for instance, would definitely help.